### ISSUE TITLE
POSIX sh case modification is undefined

### ISSUE THREAD

**Issue by kvantour** [koalaman/shellcheck#1977]
#### For bugs
- Rule Id (if any, e.g. SC1000): ??
- My shellcheck version (`shellcheck --version` or "online"): 0.7.1
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/usr/bin/sh
var="foo_BAR"
echo "${var,} ${var,,} ${var^} ${var^^}"
```

#### Here's what shellcheck currently says:
nothing

#### Here's what I wanted or expected to see:
SCxxxx: In POSIX sh, case modification is undefined.

**Comment by brother** [comment]
Maybe use `checkbashisms` instead?

`brother ~$ cat /tmp/test.sh 
#!/usr/bin/sh
var="foo_BAR"
echo "${var,} ${var,,} ${var^} ${var^^}"
brother ~$ shellcheck /tmp/test.sh
brother ~$ checkbashisms /tmp/test.sh
possible bashism in /tmp/test.sh line 3 (${parm,[,][pat]} or ${parm^[^][pat]}):
echo "${var,} ${var,,} ${var^} ${var^^}"
`

(checkbashisms is available in the devscripts package on debian based systems. current VCS location seems to be [salsa](https://salsa.debian.org/debian/devscripts/-/blob/master/scripts/checkbashisms.pl)

**Comment by koalaman** [comment]
A warning has been added, thanks!
